source/content/external-libraries.md

```diff
@@ -66,6 +66,14 @@ If your WordPress site uses Composer, consider the [PHP WkHtmlToPdf](https://git
 
 </TabList>
 
+### Unable to Generate PDF File
```

Comment:
@spheromak @winmillwill thanks for your help hunting this down! Can I please get your 👀 and 👍 on this section?